spec/features/two_factor_authentication/multiple_mfa_sign_up_spec.rb

```diff
@@ -168,7 +168,7 @@
     end
 
     it 'regenerates backup codes path if a user clicks that they need new backup codes' do
-      click_link t('two_factor_authentication.backup_codes.new_backup_codes')
+      click_link text: 'I need new backup codes'
```

Comment:
The text for this test looks needs to be hard-coded. The test will search for `I need new backup&nbsp;codes` and it will not match the parsed text.

Reply:
I think I've seen us do this elsewhere where we need to convert the HTML back to plaintext:

```suggestion
      click_link strip_tags(t('two_factor_authentication.backup_codes.new_backup_codes_html'))
```